draft-ietf-tls-sni-encryption.md

```diff
@@ -237,7 +238,7 @@ they could block it, or they could flag the users of SNI encryption for
 special treatment.
 
 In the future, it might be possible to assume that a large fraction of TLS handshakes
-use SNI encryption. If that was the case, the detection of SNI encryption would
+use SNI encryption. If that were the case, the detection of SNI encryption would
```

Comment:
My understanding is that both "was" and "were" are acceptable usage here.

Reply:
I don't know for English, but in French there are subtle differences between the two. "If it was true, ..." would imply "... but it is not", while "if it were true" would leave open the possibility that it may or may not be. So I went with Barry's suggestion. But what do I know?

Reply:
"were" works!